elements/classpower.lua

```diff
@@ -113,10 +113,17 @@ local function Update(self, event, unit, powerType)
 			mod = UnitPowerDisplayMod(ClassPowerID)
 		end
 
+		local numActive = math.ceil(mod == 0 and 0 or cur / mod)
```

Comment:
We could, although it's less future proof, what if they introduce a power with a greater accuracy, let's say 100, we'd have to keep adding fractions to that value.